### DISCUSSION TITLE
Websites built with Lightbi

### DISCUSSION THREAD

**Discussion by binokochumolvarghese** [binokochumolvarghese/lightbi-hugo#16] [Show and tell]
You can checkout my personal website which is built with Lightbi.  It is heavily customized according to personal preferences and changing constatnly over time.

Here is the [link](https://binovarghese.com/) to my webiste. Your comments and feedbacks are always welcome.

I am happy to see your websites also.  Thank you.

**Comment by WKaz** [comment]
Hi Bino! Nice theme, thank you. My test website (work in progress) built with lightbi: https://zzizp.netlify.app.

**Reply by binokochumolvarghese** [reply]
Hi, I am glad you liked the theme. 
You can use the latest theme to solve the image positioning issues in the card.

**Comment by binokochumolvarghese** [comment]
This theme is designed to help you build amazing websites with ease. I would love to hear your feedback and see your creations. Here are some of the websites I found using the Lightbi theme and feel free yo add yours.

- [Rootstalk](https://rootstalk.grinnell.edu/)
- [AIrtisan](https://airtisan-nft.com/)
- [Path of life](https://pathoflife.netlify.app/)
- [utah mountain biking](https://utahmountainbiking.com/)
- [Webcodespace](https://webcodespace.com/)
- [Atator](https://www.atator.it/)
- [Enum blog](https://www.enumhq.com/blog)
- [Ron Lunde](https://ronlunde.com/)
- [Travel to WorldWide](https://travel2worldwide.com/)
- [Z ziemi i z powietrza](https://zzizp.netlify.app/)
- [Mia Luna Chen](https://mialunachen.github.io/)
- [bahn.day](https://bahn.day/)
- [s0merset7's Blog](https://s0merset7.github.io/)
- [Tripmesh](https://blog.tripmesh.com/)
- [This is a blog](https://krokodyl.github.io/)
- [Diego Mellizo](https://diegomellizo.com/)
- [Rhea Roy Mammen](https://rheasblog.netlify.app/)
- [Культиваторний завод](https://www.savitskiy.com.ua/)

**Reply by biccio** [reply]
[PrecisionDocs](https://precisiondocs.tech/)

**Reply by Queen-Kelda** [reply]
https://fourbranches.co.uk/blog

**Reply by BlameDutchie** [reply]
Well, it took me a while to adapt to my needs and wishes, but here it is.
Thanks for the freebie!

[Blame Dutchie Photography and Words](https://blamedutchie.photography)

**Comment by Queen-Kelda** [comment]
Update, I've added a new domain, and currently using it for my Genealogy blog - Obviously using the Lightbi theme,  with Thanks to @binokochumolvarghese 

my website is https://www.whispersfrommyancestors.co.uk/
